### PR TITLE
[17.0][ADD] repair_service

### PR DESCRIPTION
**Pull request by AndreuOForgeFlow**
This module introduces repair services. It adds a new "Services" tab in the repair orders, where users can add and manage services associated with the repair. Once the repair order is confirmed and a sale order is created, these services can be automatically added as sale order lines.

### CODE REVIEW

**Review by GuillemCForgeFlow — APPROVED** [review]
LGTM! functional and code review, nice work 👍🏿

**Review by LauraCForgeFlow — APPROVED** [review]
LGTM!

File: repair_service/views/repair_views.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <record id="view_repair_order_form_inherit" model="ir.ui.view">
+        <field name="name">repair.order.form.inherit.operations - repair_service</field>
+        <field name="model">repair.order</field>
+        <field name="inherit_id" ref="repair.view_repair_order_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//notebook/page[@name='parts']" position="after">
+                <page string="Services">
+                    <field
+                        name="repair_service_ids"
+                        readonly="state == 'cancel' or state == 'done'"
```
Comment (GuillemCForgeFlow) [inline repair_service/views/repair_views.xml]:
```suggestion
                        readonly="state in ('cancel', 'done')"
```
perhaps better like this?

Comment (AndreuOForgeFlow) [reply repair_service/views/repair_views.xml]: Sure, done! :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
Code review. Interesting you can create Sales from repairs in standard Odoo17

**Review by LoisRForgeFlow — APPROVED** [review]
Code review :+1:

**Pull request merged**